src/OverlayTiming.cc

```diff
@@ -545,8 +550,9 @@ namespace overlay {
 
   void OverlayTiming::define_time_windows(const std::string &Collection_name)
   {
-    this_start = -0.25; //the integration time shall start shortly before the BX with the physics event to avoid timing problems
-                        //the value of -0.25 is a arbitrary number for the moment but should be sufficient -- corresponds to 7.5cm of flight at c
+    this_start = _DefaultStart_int; //the integration time shall start shortly before the BX with the physics event to avoid timing problems
+                        //the value of -0.25 is an arbitrary number for the moment but should be sufficient -- corresponds to 7.5cm of flight at c
+                        //But if vertex smearing is on, this may need to be shifted earlier.
```

Comment:
I think the comments could be (re)moved here. The first sentence is already part of the parameter description in the constructor. The second part (`the value of -0.25 ...`) should be moved to the declaration of `_DefaultStart_int` in the header.

Reply:
Done; thanks.